app/src/main/java/org/mozilla/focus/download/DownloadInfoManager.java

```diff
@@ -249,6 +249,7 @@ public void onQueryComplete(List downloadInfoList) {
                 }
             }
         });
+        return newId;
 
```

Comment:
check this is rowId or downloadId, then rename it

Reply:
This should be explicit already in
final long oldId = downloadId;
and
final long newId = manager.addCompletedDownload(

Reply:
Ok